### PR TITLE
Feature/issue 8

### PR DESCRIPTION
**Pull request by raynasr**
Basic home page, browse and listing screens, will be developed later on the project
Testing not required at this moment. 
Please check the code for inconsistencies and take a look at the composition of the buttons / screens.

### CODE REVIEW

File: app/src/main/java/com/studhub/app/MainActivity2.kt [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.studhub.app.ui.theme.StudHubTheme
+
+class MainActivity2 : ComponentActivity() {
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/MainActivity2.kt:21]: Activity is better named something like "HomePageActivity"

File: app/src/main/java/com/studhub/app/MainActivity2.kt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            AddListingScreen()
+        }
+        composable("Browse") {
+            BrowseScreen()
+        }
+    }
+}
+
+@Composable
+fun HomePage(onAddListingClick: () -> Unit, onBrowseClick: () -> Unit) {
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/MainActivity2.kt:61]: Don't name this "HomePage", try to split this in smaller components if necessary and just call it "Nav" for example if it contains only buttons leading to other activities.

File: app/src/main/java/com/studhub/app/MainActivity2.kt [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    StudHubTheme {
+        Text(text = "Add Listing")
+        Surface(
+            // Going to change, customize colors later.
+            modifier = Modifier.fillMaxSize(),
+            color = MaterialTheme.colors.primary
+
+        ) {}
+    }
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/MainActivity2.kt:106]: Should not even be here. This content will be inside another activity completely.

File: app/src/main/java/com/studhub/app/MainActivity2.kt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    StudHubTheme {
+        Text(text = "Browse")
+        Surface(
+            // Going to change, customize colors later.
+            modifier = Modifier.fillMaxSize(),
+            color = MaterialTheme.colors.secondaryVariant
+
+        ) {}
+    }
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/MainActivity2.kt:119]: Should not even be here. This content will be inside another activity completely.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">Bootcamp</string>
     <string name="title_activity_maps">MapsActivity</string>
+    <string name="title_activity_main2">MainActivity2</string>
```
Comment (NervzZ) [inline app/src/main/res/values/strings.xml:4]: As said in first comment, rename this to something like HomePageActivity

**Review by NervzZ — CHANGES_REQUESTED** [review]
See comments for changes, also don't forget about new lines at end of files as it's missing in some of them.

File: .idea/.name [hunk 6]
```diff
@@ -1 +1 @@
-Bootcamp
```
Comment (edouardmichelin) [inline .idea/.name:1]: Remove file

File: .idea/deploymentTargetDropDown.xml [hunk 7]
```diff
@@ -7,11 +7,11 @@
...
-            <value value="$USER_HOME$/.android/avd/Pixel_2_API_33.avd" />
+            <value value="C:\Users\User\.android\avd\Android_TV_1080p_API_33.avd" />
           </Key>
         </deviceKey>
       </Target>
     </targetSelectedWithDropDown>
-    <timeTargetWasSelectedWithDropDown value="2023-03-02T14:23:40.795216Z" />
+    <timeTargetWasSelectedWithDropDown value="2023-03-08T19:16:22.086433300Z" />
   </component>
 </project>
```
Comment (edouardmichelin) [inline .idea/deploymentTargetDropDown.xml:17]: Please use Pixel_2_API_33

File: app/build.gradle [hunk 8]
```diff
@@ -3,6 +3,7 @@ plugins {
     id 'com.google.gms.google-services'
```
Comment (edouardmichelin) [inline app/build.gradle:3]: Rebase `master` branch

**Review by edouardmichelin — CHANGES_REQUESTED** [review]
You need to rebase `master` branch
`HomePageActivity.kt` will need to be better organized i.e.: split into multiple files

**Review by NervzZ — APPROVED** [review]
LGTM now

**Review by edouardmichelin — APPROVED** [review]

**Pull request merged**